client/src/App.js

```diff
@@ -1,19 +1,40 @@
 import React from "react";
-// import LoginPage from "../src/components/Page/LoginPage";
 import GlobalStyle from "./GlobalStyle";
-// import Datepicker from "../src/components/Calendar/Calendar";
-// import ChooseYourRunPage from "../src/components/Page/ChooseYourRunPage";
-// import CreatedRunPage from "./components/Page/CreatedRunPage";
-// import RunsDonePage from "./components/Page/RunsDonePage";
+import SplashPage from "../src/components/Page/SplashPage";
+import LoginPage from "../src/components/Page/LoginPage";
+import ChooseYourRunPage from "../src/components/Page/ChooseYourRunPage";
+import CreatedRunPage from "./components/Page/CreatedRunPage";
+import RunsDonePage from "./components/Page/RunsDonePage";
 import RunSetupPage from "./components/Page/RunSetupPage";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
```

Comment:
it seems like you don't need to go up to the src level for the import ;)

Reply:
Thanks :)